packages/core/src/render3/jit/module.ts

```diff
@@ -399,7 +410,13 @@ export function resetCompiledComponents(): void {
  */
 function computeCombinedExports(type: Type<any>): Type<any>[] {
   type = resolveForwardRef(type);
-  const ngModuleDef = getNgModuleDef(type, true);
+  const ngModuleDef = getNgModuleDef(type);
+
+  // a standalone component, directive or pipe
+  if (ngModuleDef === null) {
+    return [type];
+  }
```

Comment:
```suggestion
if (isStandalone(type)) {
  return [type];
}

const ngModuleDef = getNgModuleDef(type, true);
```
nit: Personally I would think this would be easier to read and doesn't need a comment. But I can imagine it's slightly less optimal because there are a couple more lookups involved by calling `isStandalone`

Reply:
Yep - exactly.